### PR TITLE
fix: 5005+5007 carousel 3 steps post fixes

### LINKED ISSUE #5005: Carousel applied to landing
image

- follow up #4967 
- lets apply this type to all carousels
- move by 3 - more in the followed up issue
- also make the fade out animation faster, the point was make it not disappear instantly. 
- Increase the number of items on carousels; we were thinking about 30
- remove carousel position indicator (the squares under)

### LINKED ISSUE #5007: Extend amount of items in more from this collection to 60
We are showing now 20 items other from same collection under item and since we've added moving strip 3 items per click, we can extend it to 60 items so users can search more there

- https://beta.kodadot.xyz/rmrk/gallery/11407326-fa24d4027d7241d84f-PIG-PIG_GANG_995-0000000000000995

image

### PR DESCRIPTION
**Pull request by maderi21**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5005, Closes #5007
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

**Comment by exezbcz** [comment]
nice! can you please make the transition even faster, something like 2x

**Comment by exezbcz** [comment]
> 

sorry for the perfectionism, 0.2 would be perfect I think :D

**Comment by maderi21** [comment]
> > 
> 
> sorry for the perfectionism, 0.2 would be perfect I think :D

increased to 0.2

File: components/carousel/CarouselTypeNewestList.vue [hunk 1]
```diff
@@ -5,12 +5,10 @@
     :nfts="nfts"
     action-type="link"
     :link-url="`/${urlPrefix}/explore/items/?search=&sort=updatedAt_DESC`"
-    :link-text="`${$t('helper.seeMore')}`"
-    :gallery-item-carousel="false" />
+    :link-text="`${$t('helper.seeMore')}`" />
```
Comment (Jarsen136) [inline components/carousel/CarouselTypeNewestList.vue:8]:
```suggestion
    :link-text="$t('helper.seeMore')" />
```

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Comment by yangwao** [comment]
> Increase the number of items on carousels; we were thinking about 30

Let's put it on landing too 30 items for newest list and sales :)

https://user-images.githubusercontent.com/5887929/219455694-61d6fbfc-6fd1-4e1d-b9bf-72bdbd266adc.mov

**Comment by yangwao** [comment]
It seems Visited NFTs still has 10 item history too? Let's extend it to 30 :)

https://user-images.githubusercontent.com/5887929/219457141-0672cf24-f289-4506-ae02-d53350e0468d.mov

**Review by vikiival — CHANGES_REQUESTED** [review]

**Comment by vikiival** [comment]
Please do not overfetch subsquid

**Comment by maderi21** [comment]
> Increase the number of items on carousels; we were thinking about 30

didn't know this means everywhere, sorry. increased carousel fetching as follows:

landing: 
- newest 30
- latest sales 30

gallery item:
- more from collection 60
- visited 30

> Please do not overfetch subsquid

I don't know if this isn't happening now,

**Comment by yangwao** [comment]
> I don't know if this isn't happening now,

it's ok, we've talked that 60 items is not that bad, yet having resolver would be better to not overload subsquid. can be solved later if we hit performance issues on that. 

testing it, seems works well everywhere, perfect!

pay 60

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $60 @ 39.54 USD/KSM ~ 1.517 $KSM
🧗 [JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa](https://kusama.subscan.io/account/JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)
🔗 [0xa856cf35b78911cbbff6b14c26614f03ad900e0ffd23a6ae60ea62dcc06fe6d5](https://kusama.subscan.io/extrinsic/0xa856cf35b78911cbbff6b14c26614f03ad900e0ffd23a6ae60ea62dcc06fe6d5)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**